### DISCUSSION TITLE
How to consolidating Multiple Prompt Results into a Single Chat Display ?

### DISCUSSION THREAD

**Discussion by harry-stot** [FlowiseAI/Flowise#265] [Q&A]
Hello,
I'm seeking clarification on how to display the outcome of separate prompts within a single chat session. For instance, if I first prompt a summary about a company and then subsequently ask for three additional paragraphs, how can I ensure that the results for all these prompts - the summary and the three paragraphs - are shown collectively within the same chat, even though they were initiated separately? Any guidance you could provide would be greatly appreciated.

Thank you.

**Comment by HenryHengZJ** [comment]
do you have a flow that show what you are trying to achieve?

I'm lost at the part when you said subsequently ask for three additional paragraphs
